KsApi/models/graphql/GraphCommentsEnvelopeTests.swift

```diff
@@ -103,4 +104,107 @@ final class GraphCommentsEnvelopeTests: XCTestCase {
       print(error)
     }
   }
+
+  func testUpdateCommentsDecode() {
+    let dictionary: [String: Any] = [
+      "post": [
+        "id": "WzMwNDU4ODkxXQ==",
+        "comments": [
+          "edges": [
+            [
+              "node": [
+                "author": [
+                  "id": "VXNlci0xOTE1MDY0NDY3",
+                  "isCreator": true,
+                  "name": "Billy Bob",
+                  "imageUrl": "https://image.com"
+                ],
+                "authorBadges": [],
+                "body": "I have not received a survey yet either.",
+                "id": "Q29tbWVudC0zMDQ5MDQ2NA==",
+                "createdAt": 1_622_267_124,
+                "deleted": true,
+                "replies": [
+                  "totalCount": 1
+                ]
+              ]
+            ],
+            [
+              "node": [
+                "author": [
+                  "id": "VXNlci0yMDU3OTc4MTQ2",
+                  "isCreator": nil,
+                  "name": "Kate Hudson",
+                  "imageUrl": "https://image.com"
+                ],
+                "authorBadges": ["backer"],
+                "body": "I hope you guys all remembered to write in Bat Boy/Bigfoot on your ballots! Bat Boy 2020!!",
+                "id": "Q29tbWVudC0zMDQ2ODc1MA==",
+                "createdAt": 1_522_067_124,
+                "deleted": false,
+                "replies": [
+                  "totalCount": 1
+                ]
+              ]
+            ],
+            [
+              "node": [
+                "author": [
+                  "id": "VXNlci0yMTA1MDg2MzA4",
+                  "isCreator": false,
+                  "name": "Joe Smith",
+                  "imageUrl": "https://image.com"
+                ],
+                "authorBadges": ["superbacker"],
+                "body": "I haven't received my survey yet. Should I have?",
+                "id": "Q29tbWVudC0zMDQ1ODg5MQ==",
+                "createdAt": 1_622_067_114,
+                "deleted": false,
+                "replies": [
+                  "totalCount": 1
+                ]
+              ]
+            ]
+          ],
+          "pageInfo": [
+            "startCursor": "WzMwNDkwNDY0XQ==",
+            "endCursor": "WzMwNDU4ODkxXQ==",
+            "hasNextPage": true
+          ],
+          "totalCount": 61
+        ]
+      ]
+    ]
+
+    guard let data = try? JSONSerialization.data(withJSONObject: dictionary, options: []) else {
+      XCTFail("Should have data")
+      return
+    }
+
+    do {
+      let envelope = try JSONDecoder().decode(GraphCommentsEnvelope.self, from: data)
+
+      XCTAssertEqual(envelope.comments[0].id, "Q29tbWVudC0zMDQ5MDQ2NA==")
+      XCTAssertNil(envelope.comments[0].parentId)
```

Comment:
why do we need a parent id?

Reply:
I removed the `parentId` from `GraphComment` and `GraphCommentTemplates` and the app still compiled, so maybe we don't use it anywhere?

Reply:
parentId is used here`PostCommentInput`.